netebpfext/net_ebpf_ext_sock_ops.c

```diff
@@ -302,8 +302,9 @@ net_ebpf_ext_sock_ops_register_providers()
     _ebpf_sock_ops_program_info_provider_moduleid.Guid = EBPF_PROGRAM_TYPE_SOCK_OPS;
     status = net_ebpf_extension_program_info_provider_register(
         &program_info_provider_parameters, &_ebpf_sock_ops_program_info_provider_context);
-    if (status != STATUS_SUCCESS)
+    if (status != STATUS_SUCCESS) {
```

Comment:
```!NT_SUCCESS(status)```?

Reply:
Agree there are multiple instances of this in `netebpfext`, though they were not added as part of this PR. I will push another commit to fix all the instances.

Reply:
@dv-msft since this change is independent of the changes done in this PR, I am planning to have a follow-up PR (once this one gets merged) to do the cleanup in netebpfext.

Reply:
I was only referring to the ones new to _this_ PR (My apologies if I inadvertently flagged existing code) and IMO the update is trivial enough to be addressable in the PR itself :-) OTOH, I see no reason to let this trivial change delay this PR, so I'm ok with it being deferred to your next PR.

Reply:
Fixed.